sdk/metric/config.go

```diff
@@ -124,12 +124,16 @@ func WithResource(res *resource.Resource) Option {
 //
 // By default, if this option is not used, the MeterProvider will perform no
 // operations; no data will be exported without a Reader.
-func WithReader(r Reader, confs ...view.Config) Option {
+func WithReader(r Reader, views ...view.View) Option {
 	return optionFunc(func(cfg config) config {
 		if cfg.readers == nil {
-			cfg.readers = make(map[Reader][]view.Config)
+			cfg.readers = make(map[Reader][]view.View)
 		}
-		cfg.readers[r] = confs
+		if len(views) == 0 {
+			views = []view.View{{}}
```

Comment:
nit: why allocate a slice here instead of leaving it as `nil`?

Reply:
This actually allocates with an empty view.  If we didn't do that we would have to check every time we used the list of views to see if they are empty and use a default logic.